src/Tribe/Featured_Events.php

```diff
@@ -67,4 +67,26 @@ public function is_featured_query( WP_Query $query = null ) {
 
 		return (bool) $query->get( 'featured' );
 	}
+
+	/**
+	 * Indicates if 'featured' is set to a positive value either in the URL query
+	 * or the posted data (if any).
+	 *
+	 * @return bool
+	 */
+	public function featured_events_requested() {
+		$featured_request_var = tribe_get_request_var( 'featured', false );
+
+		switch ( strtolower( $featured_request_var ) ) {
+			case 'true':
```

Comment:
Could we also check for stuff like `1` and `0`? I remember us having a method that would do that check for us somewhere in our code.